pkg/dict/dict.go

```diff
@@ -32,6 +32,15 @@ func (d *Dict) Set(key, val string) {
 	d.dict[key] = val
 }
 
+// Set all global labels if already not exist
+func (d *Dict) SetAll(allKeyVals *Dict) {
+	for key, val := range allKeyVals.dict {
```

Comment:
possible NPE?

Reply:
handled. 
verified that allKeyVals can be nil, but allKeyVals.dict won't be.